cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -37,9 +37,31 @@ else()
   endif()
 endif()
 
-# Make sure we strip binaries in Release build
 if(CMAKE_BUILD_TYPE STREQUAL Release AND CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
Would it be possible to support `CMAKE_BUILD_TYPE STREQUAL MinSizeRel` as well?

Reply:
added